include/tvm/ir/memory_pools.h

```diff
@@ -40,8 +40,12 @@ struct PoolInfoNode : public Object {
    * to indicate the pool is not size restricted.
    */
   Integer size_hint_bytes;
+
+ private:
   /*! \brief The accessibility from each Target */
   Map<Target, String> target_access;  // 'rw' or 'ro'
```

Comment:
Lets remove this in the favor of array of targets

Reply:
removed